deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "KeepalivedGroup",
+          "metadata": {
+            "name": "keepalivedgroup-workers"
+          },
+          "spec": {
+            "interface": "ens3",
+            "nodeSelector": {
+              "node-role.kubernetes.io/worker": ""
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Networking, Cloud Provider
+    certified: "false"
+    containerImage: quay.io/redhat-cop/keepalived-operator:latest
+    createdAt: 02/04/2020
```

Comment:
`containerImage` and `createdAt` need to be updated

Reply:
done